mlos_bench/mlos_bench/mlos_benchd.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+mlos_bench background execution daemon.
+
+This script is responsible for polling the storage for runnable experiments and
```

Comment:
```suggestion
This script is responsible for polling the storage for runnable :py:clas:`.Experiment`s and
```

Reply:
Some minor tweaks to help make all of the docstring generation cross referencing.  Might need some tweaks.